modules/core/arrow-syntax/src/main/kotlin/arrow/syntax/function/pipe.kt

```diff
@@ -1,45 +0,0 @@
-package arrow.syntax.function
-
-infix inline fun <P1, R> P1.pipe(t: (P1) -> R): R = t(this)
```

Comment:
Are these getting migrated somewhere?

Reply:
No, we need to do a check and see what can be savaged back to core.

Reply:
We can move all these next to the definitions of `Function0` and `Function1` which I believe they are in data.